GenderPayGap.WebUI/Views/ActionHubOld/MixedResult.cshtml

```diff
@@ -0,0 +1,142 @@
+                The training might also bring to mind unhelpful stereotypes which people then act upon, or the training might make people think that the organisation has now solved its diversity problems.
+            </p>
+        </li>
+        
+        <li>
+            <h3 class="govuk-heading-s">Leadership development training</h3>
+            <p class="govuk-body">
+                Leadership development programmes aim to teach qualities including management skills and self-confidence. While there are some very small-scale studies of the effects of leadership training programmes for women, particularly in medicine and academia, there is currently no high-quality evidence that such programmes help women progress. Some people feel that these programmes imply that the women themselves are the problem.
+            </p>
+        </li>
+        
+        <li>
+            <h3 class="govuk-heading-s">Use performance self-assessments</h3>
+            <p class="govuk-body">
+                In terms of performance in the workplace, there is some evidence that women underestimate their abilities or are more conservative in their assessment of their abilities than men are. The size of this gender difference can vary depending on the type of performance people are asked to self-assess. We do not have enough evidence to know how differences in self-assessment affect women’s progression at work.
+            </p>
+        </li>
+        
+        <li>
+            <h3 class="govuk-heading-s">Diverse selection panels</h3>
+            <p class="govuk-body">
+                Having selection panels with a mix of men and women seems to help women's prospects sometimes and harm them at other times. Some studies show that the more women there are on a panel, the more likely women are to be selected for a role, while some studies find the opposite. The effect can also depend on the role being recruited for or the role of women on the committee. More research is needed to understand the conditions under which a diverse selection panel is or isn’t effective for improving gender equality.
+            </p>
+        </li>
+    </ol>
+</div>
+
+<div class="govuk-accordion" data-module="govuk-accordion" id="accordion-default">
+    <div class="govuk-accordion__section ">
+        <div class="govuk-accordion__section-header">
+            <h2 class="govuk-accordion__section-heading">
+                <span class="govuk-accordion__section-button" id="accordion-default-heading-1">
+                    References
+                </span>
+            </h2>
+        </div>
+        <div id="accordion-default-content-1" class="govuk-accordion__section-content" aria-labelledby="accordion-default-heading-1">
+            @{
+                var referenceIndex = 1;
+            }
+            <div>
+                <h4 class="govuk-heading-s">@(referenceIndex++). Unconscious bias training</h4>

```

Comment:
Could you use the new `<ol>` / `<li>` structure for the References section on this page?